common/lib/xmodule/xmodule/modulestore/courseware_index.py

```diff
@@ -83,11 +91,13 @@ def index_item_location(item_location, current_start_date):
 
             if item.has_children:
                 for child_loc in item.children:
-                    index_item_location(child_loc, current_start_date)
+                    if getattr(item, 'content_groups', None):
```

Comment:
Seems like you just pass in the value of the call to getattr; the function already accommodates when it is None., it will certainly read quite a bit better